packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -2714,17 +2726,114 @@
       }
     }
   },
-  "enterprise_search_telemetry": {
+  "apm-telemetry": {
     "dynamic": false,
     "properties": {}
   },
-  "app_search_telemetry": {
-    "dynamic": false,
-    "properties": {}
+  "apm-server-schema": {
+    "properties": {
+      "schemaJson": {
+        "type": "text",
+        "index": false
+      }
+    }
   },
-  "workplace_search_telemetry": {
-    "dynamic": false,
-    "properties": {}
+  "apm-service-group": {
+    "properties": {
+      "groupName": {
+        "type": "keyword"
+      },
+      "kuery": {
+        "type": "text"
+      },
+      "description": {
+        "type": "text"
+      },
+      "color": {
+        "type": "text"
+      }
+    }
+  },
+  "apm-custom-dashboards": {
+    "properties": {
+      "dashboardSavedObjectId": {
+        "type": "keyword"
+      },
+      "kuery": {
+        "type": "text"
+      },
+      "serviceEnvironmentFilterEnabled": {
+        "type": "boolean"
+      },
+      "serviceNameFilterEnabled": {
+        "type": "boolean"
+      }
+    }
+  },
+  "elastic-ai-assistant-prompts": {
+    "properties": {
+      "id": {
+        "type": "keyword"
+      },
+      "is_default": {
+        "type": "boolean"
+      },
+      "is_shared": {
+        "type": "boolean"
+      },
+      "is_new_conversation_default": {
+        "type": "boolean"
+      },
+      "name": {
+        "type": "keyword"
+      },
+      "prompt_type": {
+        "type": "keyword"
+      },
+      "content": {
+        "type": "keyword"
+      },
+      "updated_at": {
+        "type": "keyword"
+      },
+      "updated_by": {
+        "type": "keyword"
+      },
+      "created_at": {
+        "type": "keyword"
+      },
+      "created_by": {
+        "type": "keyword"
+      }
+    }
+  },
+  "elastic-ai-assistant-anonimization-fields": {
+    "properties": {
+      "id": {
+        "type": "keyword"
+      },
+      "field_id": {
```

Comment:
nit: Consider `field` if `field_id` represents an ES field e.g. `user.name` for consistency with other entries in `current_mappings.json` (that appear to also represent ES fields):

```json
          "field": {
            "type": "keyword"
          },
```